src/scenarios/tx_flood.py

```diff
@@ -7,24 +10,61 @@
 
 
 def cli_help():
-    return "Generate 100 blocks with 100 TXs each"
+    return "Make a big transaction mess"
 
 
 class TXFlood(WarnetTestFramework):
     def set_test_params(self):
         self.num_nodes = 1
+        self.addrs = []
+        self.threads = []
+
+    def add_options(self, parser):
+        parser.add_argument(
+            "--interval",
+            dest="interval",
+            default=10,
+            type=int,
+            help="Number of seconds between TX generation (default 10 seconds)",
+        )
+
+    def orders(self, node):
+        wallet = ensure_miner(node)
+        for address_type in ["legacy", "p2sh-segwit", "bech32", "bech32m"]:
+            self.addrs.append(wallet.getnewaddress(address_type=address_type))
+        while True:
+            sleep(self.options.interval)
+            try:
+                bal = wallet.getbalance()
+                if bal < 1:
+                    continue
```

Comment:
Can we mine a block here or top-up somehow?

Reply:
I'm leaving the block mining to other scenarios for composability. Given the randomness of this scenario, i thought it best just to skip low balance wallets. Keep in mind that even without new blocks, other nodes will be sending you bitcoin as well.